protocol/consistencychecks_test.go

```diff
@@ -3,10 +3,34 @@ package protocol
 import "testing"
 
 var (
-	uname = "alice"
-	key   = []byte("key")
+	key = []byte("key")
 )
 
+func doRequestAndVerify(d *ConiksDirectory,
```

Comment:
I think it's readable enough. If you want you could create several such functions: `createRegistreationReqAndVerify`, `createKeyLookupRequestAndVerify` (sorry for the long names, you can keep the "do*" instead)